eventstore/datadog_checks/eventstore/eventstore.py

```diff
@@ -99,19 +97,25 @@ def check(self, instance):
                 self.log.debug("Metric {} did not return a value, skipping".format(metric['json_path']))
                 self.log.info("Metric {} did not return a value, skipping".format(metric['json_path']))
 
-    def format_tag(self, name):
-        "Converts the string to snake case from camel case"
+    @classmethod
+    def format_tag(cls, name):
+        """Converts the string to snake case from camel case"""
         # https://stackoverflow.com/questions/1175208/elegant-python-function-to-convert-camelcase-to-snake-case
         s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
         return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
 
-    def walk(self, json_obj, p=[], es_paths=[]):
+    def walk(self, json_obj, p=None, es_paths=None):
         """ Walk a JSON tree and return the json paths in a list """
+        if p is None:
```

Comment:
why did you change these?

Reply:
We've made efforts to remove all mutable args

Reply:
sounds like a good change